sklearnex/preview/covariance/tests/test_covariance.py

```diff
@@ -0,0 +1,44 @@
+# ===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ===============================================================================
+
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+
+from daal4py.sklearn._utils import daal_check_version
+from onedal.tests.utils._dataframes_support import (
+    _convert_to_dataframe,
+    get_dataframes_and_queues,
+)
+
+
+@pytest.mark.parametrize("dataframe,queue", get_dataframes_and_queues())
+@pytest.mark.parametrize("macro_block", [None, 1024])
+def test_sklearnex_import_covariance(dataframe, queue, macro_block):
+    from sklearnex.preview.covariance import EmpiricalCovariance
+
+    X = np.array([[0, 1], [0, 1]])
+    X = _convert_to_dataframe(X, sycl_queue=queue, target_df=dataframe)
+    empcov = EmpiricalCovariance()
+    if daal_check_version((2024, "P", 0)) and macro_block is not None:
+        hparams = empcov.get_hyperparameters("fit")
+        hparams.cpu_macro_block = macro_block
+    result = empcov.fit(X)
+    expected_covariance = np.array([[0, 0], [0, 0]])
```

Comment:
Seems like a very basic test - would it make sense to have a bit more of a testing scope? i.e. one whose result isn't just all zeroes